### PR TITLE
fix : f-string formatting binlogstream.py L511

### LINKED ISSUE #536: Apply f-string : pymysqlreplication/binlogstream.py #L511

### PR DESCRIPTION
**Pull request by HongGeonUi**
closes #536 
I apply f-string formatting

### CODE REVIEW

**Comment by mirageoasis** [comment]
it would be better to make commit message as english since this project runs globally

**Comment by HongGeonUi** [comment]
> it would be better to make commit message as english since this project runs globally

Sorry, I recognized that, but I had mistakes.
I change all commit messages.

File: pymysqlreplication/binlogstream.py [hunk 1]
```diff
@@ -507,7 +507,7 @@ def __set_mariadb_settings(self):
         # https://mariadb.com/kb/en/5-slave-registration/
         cur = self._stream_connection.cursor()
         if self.auto_position is not None:
-            cur.execute("SET @slave_connect_state='%s'" % self.auto_position)
+            cur.execute(f'SET @slave_connect_state= "{self.auto_position}"')
```
[the remark this answers is not in the document]
Comment (HongGeonUi) [reply pymysqlreplication/binlogstream.py]: Sure, I erase the space.

**Review by sean-k1 — APPROVED** [review]
LGTM! Thanks your hard work!

**Pull request merged**